packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -403,20 +403,25 @@ export class GarbageCollector implements IGarbageCollector {
 			this.mc.logger,
 			{
 				eventName: "InitializeOrUpdateGCState",
-				details: { initialized, unrefNodeCount: this.unreferencedNodesState.size },
 			},
-			async () => {
+			async (perfEvent: {
+				end: (arg0: {
```

Comment:
This was very confusing to me.  I get it now, but I'm not sure there's value in declaring here the expected props to be sent to `perfEvent.end`.  You should be able to just do `perfEvent: PerformanceEvent` and then still pass the same details to `end` below.

Reply:
I copied this pattern from elsewhere. All I want is to log the unrefNodeCount _after_ initialization because during first load, this is always 0 otherwise. Let me try it the way you suggested.